swiftprobe/swiftprobe.go

```diff
@@ -102,7 +103,8 @@ func (p SwiftProbe) Check() error {
 		return err
 	}
 
-	err = conn.Authenticate()
+	ctx := context.Background()
```

Comment:
```suggestion
	ctx := context.TODO()
```

This is the same as context.Background() except that it shows that context must be included more globally in the project, that it's not normal.